pages/management/env-vars.md

```diff
@@ -1,74 +1,56 @@
 ---
-title: Environment Variables
+title: Service Variables
 ---
 
-# Environment Variables
+# Service Variables 
 
-Environment variables allow you to provide runtime configuration to your application which can be updated without having to modify your source code.
+Service variables allow you to provide runtime configuration to a running service without having to modify your source code. You can use them to keep secrets and other sensitive values out of your code base.
 
-You can use them to keep secrets and other sensitive values like database credentials out of your code base.
+__Note:__ Adding or modifying a service variable will cause the service to restart on any devices where the new setting is applied.
 
-__Note:__ Adding or modifying a environment will cause the application to restart on the affected device/s.
+Resin.io allows you to set service variables at two levels: fleet and device.
```

Comment:
I think this would be better as `to set service variables at both the device and fleet level.